pkg/util/users/users.go

```diff
@@ -272,6 +296,16 @@ func quoteMemberList(user spec.PgUser) string {
 	return strings.Join(memberof, ",")
 }
 
+func revokeRole(groupRole, role string, db *sql.DB) error {
```

Comment:
iirc we used to claim operator never revokes roles. Please check docs/comments to ensure it is no longer the case.

Reply:
Docs say, it does not revoke when things are removed from the manifest ;)